mne_connectivity/effective.py

```diff
@@ -5,14 +5,15 @@
 import copy
 
 import numpy as np
-from mne.utils import logger, verbose
+from mne.utils import logger
 
+from .utils import fill_doc
 from .base import SpectralConnectivity, SpectroTemporalConnectivity
 from .spectral import spectral_connectivity
 
 
-@verbose
-def phase_slope_index(data, indices=None, names=None, sfreq=2 * np.pi,
+@fill_doc
```

Comment:
This has a `verbose` kwarg so this should stay `verbose`:
```suggestion
@verbose
@fill_doc
```
The `@verbose` kwarg does two things: fills doc, then also sets the `logger.level` if `verbose is not None`. If you change this to just `fill_doc`, you lose the second part. By doing `@fill_doc` then `@verbose`, you should get both.